test/manual/api/utils.sh

```
@@ -0,0 +1,167 @@
+    for conn_id in $CREATED_CONNECTION_IDS; do
+      # Try with a direct DELETE request to the connection endpoint
+      make_request DELETE "$API_BASE_URL/api/maps/$MAP_SLUG/connections/$conn_id" > /dev/null 2>&1 || true
+    done
+  fi
+  
+  # Then delete systems
+  if [ -n "$CREATED_SYSTEM_IDS" ]; then
+    echo "Cleaning up systems..."
+    
+    # First try batch delete if we have multiple systems
+    if [ $(echo "$CREATED_SYSTEM_IDS" | wc -w) -gt 1 ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)